src/main/java/gregtech/api/util/CTRecipeHelper.java

```diff
@@ -101,6 +102,92 @@ public static String getRecipeRemoveLine(RecipeMap<?> recipeMap, Recipe recipe)
+            for (GTRecipeInput ci : recipe.getInputs()) {
+                String ingredient = getCtItemString(ci);
+                if (ingredient != null)
+                    builder.append(ingredient);
+            }
+            builder.delete(builder.length() - 2, builder.length())
+                    .append("])");
+        }
+
+        if (recipe.getFluidInputs().size() > 0) {
+            builder.append(".fluidInputs(");
+            builder.append("[");
+            for (GTRecipeInput fluidStack : recipe.getFluidInputs()) {
+
+                builder.append("<liquid:")
+                        .append(fluidStack.getInputFluidStack().getFluid().getName())
+                        .append(">");
+
+                if (fluidStack.getAmount() > 1) {
+                    builder.append(" * ")
+                            .append(fluidStack.getAmount());
+                }
+
+                builder.append(", ");
+            }
+            builder.delete(builder.length() - 2, builder.length())
+                    .append("])");
+        }
+
+        if (recipe.getOutputs().size() > 0) {
+            builder.append(".outputs(");
+            builder.append("[");
+            for (ItemStack itemStack : recipe.getOutputs()) {
+                String itemId = getMetaItemId(itemStack);
+                if (itemId != null) {
+                    builder.append("<metaitem:")
+                            .append(itemId)
+                            .append(">");
+                } else {
+                    builder.append("<")
+                            .append(itemStack.getItem().getRegistryName().toString())
+                            .append(":")
+                            .append(itemStack.getItemDamage())
+                            .append(">");
+                }
+
+                if (itemStack.serializeNBT().hasKey("tag")) {
+                    String nbt = NBTConverter.from(itemStack.serializeNBT().getCompoundTag("tag"), false).toString();
+                    if (nbt.length() > 0) {
+                        builder.append(".withTag(").append(nbt).append(")");
+                    }
+                }
+            }
+            builder.delete(builder.length() - 2, builder.length())
+                    .append("])");
+        }
+
+        if (recipe.getFluidOutputs().size() > 0) {
+            builder.append(".fluidOutputs(");
+            builder.append("[");
+            for (FluidStack fluidStack : recipe.getFluidOutputs()) {
+                builder.append("<liquid:")
+                        .append(fluidStack.getFluid().getName())
+                        .append(">");
+                if (fluidStack.amount > 1) {
+                    builder.append(" * ")
+                            .append(fluidStack.amount);
+                }
+            }
+            builder.delete(builder.length() - 2, builder.length())
+                    .append("])");
+
+        }
+
+        builder.append("....");
```

Comment:
What is the purpose of adding this to the end?

Reply:
the string output shown does not include EU/t, Chanced Outputs or other recipe properties